src/Agent.Worker/Build/TeeTFCommandManager.cs

```diff
@@ -257,7 +257,7 @@ private string FormatArguments(FormatFlags formatFlags, params string[] args)
 
             if (!formatFlags.HasFlag(FormatFlags.OmitLogin))
             {
-                formattedArgs.Add($"-login:_,{accessToken}");
+                formattedArgs.Add($"-jwt:{accessToken}");
```

Comment:
I assume this is tested.  Did we try VSTS and on-prem (QU2 is interesting to try for now)